renderer/reducers/app.js

```diff
@@ -105,6 +106,16 @@ export const terminateApp = () => async dispatch => {
   dispatch(send('terminateAppSuccess'))
 }
 
+/**
+ * openPreferences - IPC handler for 'openPreferences' message.
+ */
+export const openPreferences = () => (dispatch, getState) => {
+  const modals = modalSelectors.getModalState(getState())
+  if (!modals.find(m => m.type === 'SETTINGS')) {
```

Comment:
could be a good idea to add this kind of guard to `openModal` itself to prevent this same issue for other modals

Reply:
Good idea. I've moved this guard back to `openModal`